net/unbound/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 PortGroup           compiler_blacklist_versions 1.0
 
 name                unbound
-version             1.9.5
+version             1.9.6
+revision	    1
```

Comment:
The indentation on the revision number here is wrong, and it should be revision 0 because you have updated the version number.